resources/fe-test/src/test.js

```diff
@@ -56,6 +56,12 @@ async function clickMakeMove(driver, who) {
 async function firefox_start_and_first_move(driver, baseUrl) {
   await driver.get(baseUrl);
 
+  await selectSimulator(driver);
+
+  await wait(driver, 5.0);
```

Comment:
Can you use `wait.until()` here, or is that too much upheaval to select a changed element, etc ?